apps/api/src/common/trim.pipe.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { Injectable, PipeTransform, ArgumentMetadata } from '@nestjs/common';
+
+@Injectable()
+export class TrimPipe implements PipeTransform {
+  private isObj(obj: any): boolean {
+    return typeof obj === 'object' && obj !== null;
+  }
+
+  private trim(values: any) {
+    Object.keys(values).forEach(key => {
+      if (this.isObj(values[key])) {
+        values[key] = this.trim(values[key]);
+      } else {
+        if (typeof values[key] === 'string') {
+          values[key] = values[key].trim();
+        }
+      }
+    });
+    return values;
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Called it!  😂 
All good though, having any here makes sense.

Reply:
It was a matter of time :-D